plugins/cliconf/iosxr.py

```diff
@@ -94,7 +95,7 @@ def get_device_info(self):
                     device_info["network_os_model"] = match.group(1)
                     break
 
-            match = re.search(r"^(.+) uptime", data, re.M)
+            match = re.search(r"hostname\s(\S+)$", hostname, re.M)
```

Comment:
We should have an unit test that validates that this change does not break this method for older versions.